application_store_api/namespace_fund/data_store/data.py

```diff
@@ -0,0 +1,191 @@
+from dateutil import parser as date_parser
+
+initial_fund_store_state = {
```

Comment:
I wonder if these two dicts could share a naming convention like "initial_<name of data store>_state" or "initial_<name of data store>"?

Reply:
💯 This would be nice to see, should be easy to do in vscode - use rename symbol